schema/fields_test.go

```diff
@@ -62,6 +62,10 @@ func TestFieldBuilder_Build(t *testing.T) {
 				builder:  &FieldBuilder{FieldName: "test", Type: "integer", Primary: &boolTrue},
 				expError: nil,
 			},
+			{
+				builder:  &FieldBuilder{FieldName: "_test", Type: "integer"},
```

Comment:
Is this tests new behavior?

Reply:
no, it is just to test existing behavior. We already allow field name with `_`